.github/workflows/docker-build-static.yml

```diff
@@ -453,26 +484,39 @@ jobs:
           # list of Docker images to use as base name for tags
           images: |
             nitnelave/lldap
-          # generate Docker tags based on the following events/attributes
+            lldap/lldap
+            ghcr.io/lldap/lldap
+          # generate Docker tags based on the following events/attributes, with -alpine or -debian
+          # vX-debian / -alpine
+          # vX.Y-debian / -alpine
+          # vX.Y.Z-debian / -alpine
+          # Suffix override on latest and stable tag, stable stable-debian, latest, latest-debian
+          flavor: |
+            latest=false
+            suffix= ${{ matrix.suffix }}
           tags: |
-            type=ref,event=branch
             type=ref,event=pr
-            type=semver,pattern={{version}}
-            type=semver,pattern={{major}}.{{minor}}
-            type=semver,pattern={{major}}
-            type=sha
-
-      - name: parse tag
-        uses: gacts/github-slug@v1
-        id: slug
-
-      - name: Login to Docker Hub
+            type=semver,pattern=v{{version}}
+            type=semver,pattern=v{{major}}
+            type=semver,pattern=v{{major}}.{{minor}}
+            type=raw,value=latest,enable={{ is_default_branch }},suffix=${{ matrix.noflavour }}
+            type=raw,value=stable,enable=${{ startsWith(github.ref, 'refs/tags/v') }},suffix=${{ matrix.noflavour }}
+
+      # Docker login to nitnelave/lldap and lldap/lldap
+      - name: Login to Nitnelave Docker Hub
         if: github.event_name != 'pull_request'
         uses: docker/login-action@v2
         with:
           username: ${{ secrets.DOCKERHUB_USERNAME }}
           password: ${{ secrets.DOCKERHUB_TOKEN }}
 
+      - name: Login to LLDAP Docker Hub
+        if: github.event_name != 'pull_request'
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_LLDAP_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_LLDAP_TOKEN }}
+
```

Comment:
Prepare to login to both repo for `lldap/lldap` and `nitnelave/lldap`.

Reply:
`lldap` is an organization on DockerHub too, so you can't log in as it. I *think* that being logged in as nitnelave is enough (we already pushed an image to lldap/lldap).

Reply:
On update repo part
```

      - name: Update repo description
        if: github.event_name != 'pull_request'
        uses: peter-evans/dockerhub-description@v3
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_PASSWORD }}
          repository: nitnelave/lldap

      - name: Update lldap repo description
        if: github.event_name != 'pull_request'
        uses: peter-evans/dockerhub-description@v3
        with:
          username: ${{ secrets.DOCKERHUB_USERNAME }}
          password: ${{ secrets.DOCKERHUB_PASSWORD }}
          repository: lldap/lldap
```
Is this working correctly? Same secrets?